src/MoBi.Core/Domain/Extensions/BuildingBlockExtensions.cs

```diff
@@ -71,6 +71,9 @@ private static bool modulesShareName(Module module1, Module module2)
       {
          return module1 != null && module2.IsNamed(module1.Name);
       }
+
+      public static bool IsPkSimModule(this IBuildingBlock buildingBlock) =>
+         buildingBlock.Module?.IsPKSimModule ?? false;
```

Comment:
somehow line 76 crashes in several tests. probably should check if `buildingBlock `is not null as well